src/main/java/jenkins/branch/OrganizationFolder.java

```diff
@@ -174,6 +174,7 @@ public void complete() throws IllegalStateException, InterruptedException {
                                 MultiBranchProject<?, ?> project;
                                 try {
                                     project = factory.createProject(OrganizationFolder.this, projectName, sources, Collections.<String,Object>emptyMap(), listener);
+                                    project.setOrphanedItemStrategy(getOrphanedItemStrategy());
```

Comment:
Propagated when the project is created. We have to do the same when the `OrganizationFolder` is updated, _propagate again_.